src/config.js

```diff
@@ -10,218 +10,96 @@
 'use strict';
 
 const fs = require('fs');
-const path = require('path');
+require('dotenv').config({ path: 'local.env' });
```

Comment:
what does dotenv give us that we dont have already with docker (docker supports env files)?

Reply:
It's convenient during development when you run app directly on local machine without docker.